src/inputHelper.ts

```diff
@@ -9,4 +9,9 @@ export const runQualityChecks = core.getInput("run-quality-checks");
 
 export function isRunQualityChecksEnabled(): boolean {
     return runQualityChecks.toLowerCase() === "true";
+}
+
+export function validateRequiredInputs() {
+    if (!imageName)

```

Comment:
Why does this have only single input validation? :P

Reply:
Only required inputs need to be validated. And there's just one required input